lib/solaar/ui/window.py

```diff
@@ -578,16 +562,15 @@ def _update_receiver_panel(receiver, panel, buttons, full=False):
 	assert receiver
 
 	devices_count = len(receiver)
-	if receiver.max_devices > 1:
-		if devices_count == 0:
-			panel._count.set_markup(_UNIFYING_RECEIVER_TEXT[0] % receiver.max_devices)
-		else:
-			panel._count.set_markup(_UNIFYING_RECEIVER_TEXT[1] % (devices_count, receiver.max_devices))
-	else:
-		if devices_count == 0:
-			panel._count.set_markup(_NANO_RECEIVER_TEXT[0])
-		else:
-			panel._count.set_markup(_NANO_RECEIVER_TEXT[1])
+	
+	paired_text = gettext.gettext('No device paired.') if devices_count == 0 else gettext.ngettext('One paired device.', '%(devices_count)s paired devices.', devices_count) % { 'devices_count': devices_count }
+		
+	if(receiver.max_devices > 0):
+		paired_text += gettext.ngettext('\n\n<small>Up to %(max_devices_count)s device can be paired to this receiver.</small>', '\n\n<small>Up to %(max_devices_count)s devices can be paired to this receiver.</small>', receiver.max_devices) % { 'max_devices_count': receiver.max_devices }
```

Comment:
Since the previous translation was a  set of separate strings, I am leaning towards separate HTML-less strings as well. If you think that mixing markup is fine, then you can do the change anyway I guess